doc/CodeDocumentation.dox

```diff
@@ -221,6 +221,17 @@ namespace mfem {
  * - <a class="el" href="pdiffusion_8cpp_source.html">DPG Diffusion example</a>: DPG formulation for the diffusion problem
  * - <a class="el" href="pmaxwell_8cpp_source.html">DPG Maxwell example</a>: DPG formulation for the indefinite Maxwell problem
  * - <a class="el" href="lor__elast_8cpp_source.html">LOR Elasticity</a>: solve linear elasticity with LOR preconditioning on GPUs
+ * - <a class="el" href="nurbs__ex1_8cpp_source.html">NURBS Example 1</a>: the Laplace problem using a NURBS-basis
+ * - <a class="el" href="nurbs__ex1p_8cpp_source.html">NURBS Example 1p</a>: parallel implementation of the Laplace problem using a NURBS-basis
+ * - <a class="el" href="nurbs__patch__ex1_8cpp_source.html">NURBS Example 1 Patch integration</a>: the Laplace problem using a NURBS-basis and patch integration
+ * - <a class="el" href="nurbs__ex3_8cpp_source.html">NURBS Example 3</a>: definitive Maxwell problem using a NURBS-based H(curl) elements
+ * - <a class="el" href="nurbs__ex5_8cpp_source.html">NURBS Example 5</a>: mixed Darcy problem using a NURBS-based H(div) elements
+ * - <a class="el" href="nurbs__ex11p_8cpp_source.html">NURBS Example 11p</a>: parallel implementation of the Laplace eigenvalue problem using a NURBS-basis
+ * - <a class="el" href="nurbs__ex24_8cpp_source.html">NURBS Example 24</a>: test problems for NURBS-based H_1, H(curl) and H(div) elements
+ * - <a class="el" href="nurbs__solenoidal_8cpp_source.html">NURBS Divergence-free</a>: solve a solenoidal vector projection with NURBS-based H(div) elements
+ * - <a class="el" href="nurbs__curveint_8cpp_source.html">NURBS Interpolation</a>: NURBS interpolation of given geometry
+ * - <a class="el" href="nurbs__naca__cmesh_8cpp_source.html">NURBS NACA Mesher</a>: generate NURBS based mesh around a NACA foil
+ * - <a class="el" href="nurbs__printfunc_8cpp_source.html">NURBS Printer</a>: print the NURBS-basis
```

Comment:
Maybe it is better to separate these into their own section as e.g. the "<H4>SUNDIALS Examples</H4>" above?